assets/dev/js/editor/editor-base.js

```diff
@@ -28,7 +28,7 @@ export default class EditorBase extends Marionette.Application {
 
 	helpers = require( 'elementor-editor-utils/helpers' );
 	imagesManager = require( 'elementor-editor-utils/images-manager' ); // TODO: Unused.
-	debug = require( 'elementor-editor-utils/debug' );
+	debug = elementorCommon.debug;
```

Comment:
1. Add deprecation to this property (use getter with deprecation notice)
2. Change all internal cases to use `elementorCommon.debug`